### ISSUE TITLE
Report show's failed on form header even though the session is successful

### ISSUE THREAD

**Issue by gveater** [VeeamHub/powershell#10]
![veeamcapture](https://cloud.githubusercontent.com/assets/23031506/19646366/16e3a094-99c7-11e6-89bd-a9fdd5ac742e.PNG)
The picture shows that the top of the form is the color red and reads error, even though all of the backups in this job are successful.  I looked through and it seems to be the same on every job that had a recent failure.  I have the max variable set to 1 so that I only get the results of the last session, but it looks like it is pulling the info for the top of the form from all of the job session history.

**Comment by poulpreben** [comment]
Hi @gveater. Can you please double check whether the job failed on a synthetic operation, such as incremental backup file merge or synthetic full creation? This is typically the reason for such reports.

**Comment by poulpreben** [comment]
Adding author @tdewin as well.

**Comment by gveater** [comment]
We are currently testing on how to move our copy to external media jobs so I have had multiple failures for various reasons on this job.  The last two failure were unexpected failures, from me stopping the job.  Before that there were a lot of media not found.  This itself is a Backup Copy job.  From my understanding it does a synthetic full with whatever backup is currently on that HDD.

**Comment by tdewin** [comment]
Confirmed, this can be checked by running limited code below. Backup Copy Job (or backupsync) return status None

```powershell
$jobtype = "BackupSync"
#$jobtype = "Backup"
$Jobs = @(Get-VBRJob | ? { $_.JobType -ieq $jobtype }) | Sort-Object -Property Name

$allsessions = Get-VBRBackupSession | ? { $_.jobtype -ieq $jobtype } 
$allorderdedsess = $allsessions | Sort-Object -Property CreationTimeUTC -Descending  

foreach ($Job in $Jobs) {
     $lastsession = $allorderdedsess | ? { $_.jobname -eq $Job.Name } | select -First 1
     if ($lastsession -ne $null) {
            write-host ("Name : {0} Last Result : {1}  Date : {2}" -f $Job.Name,$lastsession.Result,$lastsession.CreationTime)
     } else {
           write-host ("Could not find Session for {0}" -f $Job.Name)
     }
} 
```

**Comment by tdewin** [comment]
Update is in pull request now. Code manually evaluates "Success". Case can be closed when PR is committed.

**Comment by poulpreben** [comment]
Closing. Fixed in PR #11. Thanks @tdewin and @gveater.
